pkg/streamingpromql/types/limiting_pool.go

```diff
@@ -75,9 +87,34 @@ var (
 		}),
 		HistogramPointerSize,
 		true,
+		mangleHistogram,
 	)
 )
 
+func mangleHistogram(h *histogram.FloatHistogram) *histogram.FloatHistogram {
+	if h == nil {
+		return nil
+	}
+
+	h.ZeroCount = 12345678
+	h.Count = 12345678
+	h.Sum = 12345678
```

Comment:
(nit) it's *very* unlikely, but if we write a test with these values elsewhere the mangle won't work. Maybe for just one of these (eg Sum) we multiply it or similar.
```suggestion
	h.Sum = h.Sum * 9.87654321
```

Reply:
I'm keen to keep this as simple as possible and to make the resulting histogram easy to identify. I think the likelihood of us using a histogram with these values elsewhere and also having a bug that is then hidden because of that is very, very unlikely, as you say.

Reply:
Fair enough, I don't hold this strongly.